test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -635,30 +635,20 @@ public void testGetDocContent() throws Exception {
     assertEquals(metadata, response.getMetadata());
   }
 
-  @Test
-  public void testClobColumnAsMetadata() throws Exception {
-    String content = "Hello World";
-    executeUpdate("create table data(ID int, CONTENT clob)");
-    String sql = "insert into data(ID, CONTENT) values (1, ?)";
-    try (PreparedStatement ps = getConnection().prepareStatement(sql)) {
-      ps.setString(1, content);
-      assertEquals(1, ps.executeUpdate());
-    }
-
+  private void testColumnTypeMetadata(String content) throws Exception {
```

Comment:
We aren't doing this in the DB tests at the moment. I'd like to remove some of the config cruft, but in particular, having the DB inserts, exercising the OUT, and the assertions in each is by design.